letsdrill.rb

```diff
@@ -1,24 +1,37 @@
 def get_letter_grade(integer)
- 
+
   #Put your code here!
+  case integer
+  when 90..100
+    return "A"
```

Comment:
Rubyists tend to avoid explicit `return`. This method should still work without it. A la:

```ruby
when 90..100
  "A"
```